tests/Jellyfin.Networking.Tests/NetworkParseTests.cs

```diff
@@ -219,77 +148,18 @@ public void IpV6SubnetMaskMatchesValidIpAddress(string netMask, string ipAddress
         [InlineData("2001:db8:abcd:0012::0/128", "2001:0DB8:ABCD:0012:0000:0000:0000:0001")]
         public void IpV6SubnetMaskDoesNotMatchInvalidIpAddress(string netMask, string ipAddress)
         {
-            var ipAddressObj = IPNetAddress.Parse(netMask);
-            Assert.False(ipAddressObj.Contains(IPAddress.Parse(ipAddress)));
+            var split = netMask.Split("/");
+            var mask = int.Parse(split[1], CultureInfo.InvariantCulture);
+            var ipa = IPAddress.Parse(split[0]);
+            var ipn = new IPNetwork(ipa, mask);
+            Assert.False(ipn.Contains(IPAddress.Parse(ipAddress)));
         }
 
         [Theory]
-        [InlineData("10.0.0.0/255.0.0.0", "10.10.10.1/32")]
-        [InlineData("10.0.0.0/8", "10.10.10.1/32")]
-        [InlineData("10.0.0.0/255.0.0.0", "10.10.10.1")]
-
-        [InlineData("10.10.0.0/255.255.0.0", "10.10.10.1/32")]
-        [InlineData("10.10.0.0/16", "10.10.10.1/32")]
-        [InlineData("10.10.0.0/255.255.0.0", "10.10.10.1")]
-
-        [InlineData("10.10.10.0/255.255.255.0", "10.10.10.1/32")]
-        [InlineData("10.10.10.0/24", "10.10.10.1/32")]
-        [InlineData("10.10.10.0/255.255.255.0", "10.10.10.1")]
-
-        public void TestSubnetContains(string network, string ip)
-        {
-            Assert.True(IPNetAddress.TryParse(network, out var networkObj));
-            Assert.True(IPNetAddress.TryParse(ip, out var ipObj));
-            Assert.True(networkObj.Contains(ipObj));
-        }
-
-        [Theory]
-        [InlineData("192.168.1.2/24,10.10.10.1/24,172.168.1.2/24", "172.168.1.2/24", "172.168.1.2/24")]
```

Comment:
Not necessary?

Reply:
Same as above - those tests are not necessary if we use the .NET Objects for IPs and Subnets.